From f939cffdac08fb40eac0e633a8d58965b9ee275a Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Tue, 18 Aug 2015 15:05:11 +0200
Subject: [PATCH] test/TEST-04-FULL-SYSTEMD/create-root.sh: modprobe btrfs

why so ever... it used to load automatically

(cherry picked from commit 65977ef1d107c8741d9f2633b8618aa3aee2a49d)
---
 test/TEST-04-FULL-SYSTEMD/create-root.sh | 1 +
 1 file changed, 1 insertion(+)

diff --git a/test/TEST-04-FULL-SYSTEMD/create-root.sh b/test/TEST-04-FULL-SYSTEMD/create-root.sh
index 5972bd3..0a66257 100755
--- a/test/TEST-04-FULL-SYSTEMD/create-root.sh
+++ b/test/TEST-04-FULL-SYSTEMD/create-root.sh
@@ -4,6 +4,7 @@ for x in 64-lvm.rules 70-mdadm.rules 99-mount-rules; do
     > "/etc/udev/rules.d/$x"
 done
 rm -f -- /etc/lvm/lvm.conf
+modprobe btrfs
 udevadm control --reload
 set -e
 # save a partition at the beginning for future flagging purposes
